### PR TITLE
Edit trials

### PR DESCRIPTION
**Pull request by eudondian**
I edited the HTML

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
     <header id="header" >
       <img id="header-img"src="https://res.cloudinary.com/ds5gvclyz/image/upload/v1647827635/chiffon_f83wan.jpg" width="50" height="50"
```
Comment (eudondian) [inline index.html:13]: The Image should have an anchor <a> tag so that your website will be more professional.

File: index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
 </header>
 <div class="container">
       
-  <div class="containers">
+  <div class="container">
```
Comment (eudondian) [inline index.html:25]: The  semantic tag (<divs>) are meant to be inside the section tags

File: index.html (hunk 1 above)
Comment (eudondian) [inline index.html:13]: The image should be wrapped with an anchor tag to make your website more profesional

File: index.html (hunk 2 above)
Comment (eudondian) [inline index.html:25]: The  semantic tag divs are meant to be inside the section tags. Please rearrange it.

**Review by eudondian — COMMENTED** [review]
# Changes  required 

Kindly effect the required changes on comment section and submit for re-review .

Thank you

**Review by vechicin — APPROVED** [review]
# Hi @eudondian  ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good use of inline comments
- [x] Clear, polite comments

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**